README.md

```diff
@@ -1,73 +1,96 @@
-# Role Name
+# crypto_policies
 
-A template for an ansible role which configures some GNU/Linux subsystem or
-service. A brief description of the role goes here.
+This Ansible role manages system-wide crypto policies.
+
+This concept is well adopted since Red Hat Enterprise Linux 8 and in Fedora.
 
 ## Requirements
 
-Any pre-requisites that may not be covered by Ansible itself or the role should
-be mentioned here. For instance, if the role uses the EC2 module, it may be a
-good idea to mention in this section that the `boto` package is required.
+The system-wide crypto policies are implemented and tested on RHEL 8/CentOS 8
+and Fedora.
+
+The role might work elsewhere, but might need enabling in [vars/](vars/).
 
 ## Role Variables
 
-A description of all input variables (i.e. variables that are defined in
-`defaults/main.yml`) for the role should go here as these form an API of the
-role.
+Unconfigured, this role will just report system status as described in the
+following section.
 
-Variables that are not intended as input, like variables defined in
-`vars/main.yml`, variables that are read from other roles and/or the global
-scope (ie. hostvars, group vars, etc.) can be also mentioned here but keep in
-mind that as these are probably not part of the role API they may change during
-the lifetime.
+ * `crypto_policies_policy`
 
-Example of setting the variables:
+Use this variable to specify the desired crypto policy on the target system,
+which can be either the base policy or a base policy with policy submodule(s)
+as accepted by the `update-crypto-policies` tool.For example `FUTURE` or
+`DEFAULT:NO-SHA1:GOST`. The specified base policy and policy submodule(s)
+must be available on the target system.
 
-```yaml
-template_foo: "oof"
-template_bar: "baz"
-```
+The default value is `false` meaning the configuration is not changed and
+the role will just collect the facts below.
+
+ * `crypto_policies_reload`
+
+By default (`true`), updating crypto policies forces reload of some of
+the daemons affected by crypto policies in the system. Setting `false`
+prevents this behavior and is helpful if the role is executed during system
+enrollment or some other follow-up tasks is expected to do it later.
+
+ * `crypto_policies_reboot`
+
+Crypto policies can not know all the custom applications using crypto
+libraries that are affected by change of crypto policies. By default (`true`),
+the machine is rebooted after applying changes. To prevent the reboot,
```

Comment:
All of the other roles are "opt-in" - the role will not reboot the machine unless explicitly told to do so.  I think crypto policies should be the same - the default should be `crypto_policies_reboot: false`

Reply:
In that case, we should do it the same way. Let me change it in this PR.